streams/src/main/java/org/apache/kafka/streams/state/internals/MemoryLRUCache.java

```diff
@@ -29,6 +29,16 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * An in-memory LRU cache store based on HashSet and HashMap.
+ *
+ * Note that array typed keys may result in incorrect behavior.
```

Comment:
I'd add: "If you are working on byte arrays, for example, you may want to wrap them with the `Bytes` class."